pkg/cli/cmd/radinit/display_test.go

```diff
@@ -115,7 +118,7 @@ func Test_summaryModel(t *testing.T) {
 			},
 		}
 
-		expected := "You've selected the following:\n" +
+		expected := "\rYou've selected the following:\n" +
```

Comment:
Now the behavior of the library is that it adds `\r`: https://github.com/charmbracelet/bubbletea/pull/1008/files#diff-eec1160b5c25625c859865f3ea86e45e76521a2dd877c78a508183b968646f7dR235.